docs/cloud.md

```diff
@@ -0,0 +1,239 @@
+to the publicly visible IP of the machine where you are running these commands.
+
+See the [eksctl cluster access](https://eksctl.io/usage/vpc-cluster-access/)
+documentation to learn more.
+:::
+
+:::tip
+`eksctl` allows additional cluster setup and access configuration beyond what
+we've shown here. Check out the [configuration file
+schema](https://eksctl.io/usage/schema/) to learn more.
+:::
+
+To verify that the EKS cluster is running, run:
+
+```
+kubectl get nodes
+```
+
+You should see a three node cluster, all with the status `Ready`.
+
+## Cluster ingress
+
+To allow external access to Feldera and other services running inside the EKS
+cluster, we recommend setting up a Kubernetes Ingress controller. This is
+useful to allow access to the Feldera Web Console or to issue REST API calls
+directly. 
+
+:::tip
+When you install an ingress controller, it uses an AWS load balancer to expose
+an IP address via a generated DNS name for external access. By default,
+creating ingress controllers leads to both an internal and external load
+balancer being created, with the external one being prefered to expose
+services. Most Feldera Cloud users should need to expose Feldera services only
+over an internal load balancer.
+:::
+
+To install an ingress controller where the external load balancer is disabled,
+first create the following file:
+
+```
+# File name: ingress-controller-config.yaml
+
+controller:
+  ingressClassByName: true
+
+  ingressClassResource:
+    name: nginx
+    enabled: true
+    default: false
+    controllerValue: "k8s.io/ingress-nginx-internal"
+
+  service:
+    # Disable the external LB
+    external:
+      enabled: false
+
+    # Enable the internal LB.
+    internal:
+      enabled: true
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-internal: "true"
+        service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp
+        service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: 'true'
+        service.beta.kubernetes.io/aws-load-balancer-type: nlb
+```
+
+Then use Helm to install the nginx Ingress controller:
+
+```
+kubectl create ns ingress-nginx-internal
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+helm repo update
+helm upgrade --install ingress-nginx-internal ingress-nginx/ingress-nginx \
+  -n ingress-nginx-internal \
+  -f ingress-nginx.yaml
+```
+ 
+## Installing Feldera Cloud
+
+To install Feldera Cloud, first get the Helm package and container images that
+are provided with our license. Please contact support@feldera.com if you do not
+already have these artifacts for your install.
+
+Next, let's create a Kubernetes namespace to deploy Feldera Cloud in.
+
+```
+kubectl create ns feldera
+```
+
+Next, let's use Helm to install Feldera. We will create a Helm release named 
+`feldera` in the `feldera` namespace we just created.
+
+```
+helm upgrade --install feldera ./helm/feldera \
```

Comment:
Do we specify internal / external Kafka as an option here?

Reply:
No we don't. This brings up Feldera Cloud. Adding connectors will be a separate bit of documentation.

Reply:
> No we don't. This brings up Feldera Cloud. Adding connectors will be a separate section.

If a customer does not have a Kafka service how will the Debezium connector work?

Reply:
The documentation to set up a Debezium connector will also include instructions to bring up Kafka + Kafka connect.

Reply:
> The documentation to set up a Debezium connector will also include instructions to bring up Kafka + Kafka connect.

I am not sure the following exchange is a good user experience, irrespective of what other companies do:

1. data ops users needs data from mysql source
2. ask sys ops user to go to feldera k8s and start kafka service and configure it for capacity, perf, etc.
3. ask sys ops user to start kafka connect server
4. ask sys ops user to configure debezium to point to a specific mysql source, including connectivity creds
4. ask sys ops to validate mysql -> debezium -> kafka works
5. hand over to data ops user to configure an input connector instance
6. repeat, debug, redo in case of errors, failure, etc.

Reply:
It's on the roadmap to figure out this UX (main blocker being that our APIs currently do not provision/manage any new services). Until then, some of these will be handled out of band. We need some experience doing it the manual way to understand what and what not to automate.